### DISCUSSION TITLE
Update a maintenance window

### DISCUSSION THREAD

**Discussion by JLLormeau** [Dynatrace/dynatrace-configuration-as-code#321] [Archive] [answered]
Hello, Is there a way to update a maintenance window? With monaco, I only succeed to create maintenance window, but not find how to update it. . Regards JLL

**Comment by tatomir146** [comment, marked as answer]
All configs are automatically created/updated. If config exists, it's updated, otherwise it will be created.

**Reply by JLLormeau** [reply]
Hello, I have found my issue : 
in the json file, I used 
{
  "name": "**Ondemand**:{{ .name }}"
}
with this configuration, the update doesn't work. 
and it works fine  if I only use : 
{
  "name": "{{ .name }}"
}
Thanks a lot, 
Regards JLL
